cdbot/cogs/cyber.py

```diff
@@ -346,6 +346,19 @@ def __init__(self, bot: Bot):
 
         await self.countdown('5th March 2019', 'CyberStart Essentials', ctx)
 
+    @command()
+    async def hundered(self, ctx: Context):
+        """
+        Gets the number of 100% and true 100% users
+        """
+
+        game_r = utils.get(ctx.guild.roles, name='100 Percent')
+        true_r = utils.get(ctx.guild.roles, name='True 100%')
```

Comment:
This should be using an id stored in `constants.py`, rather than a hard-coded name. This is both bad practice and could potentially break if role names change or there are multiple roles with the same name, as has been known to happen.

Reply:
True 100%: 552536443948367903
100%: 532654673270669313

Reply:
Fixed